vibes/vibed-ui/examples/demo_basic.cpp

```
@@ -0,0 +1,65 @@
+    root->addChild(button);
+
+    ui::ViewTree tree;
+    tree.setRoot(root);
+
+    bool shouldExit = false;
+    for (int frame = 0; frame < 240 && !shouldExit; ++frame) {
+        if (platform::activePlatform() != nullptr) {
+            platform::activePlatform()->pumpEvents();
+        }
+
+        core::Event event;
```

Clang (14.0.6)

[warning] variable 'event' is not initialized

(cppcoreguidelines-init-variables)